etna/transforms/timestamp/holiday.py

```diff
@@ -27,8 +65,9 @@ class HolidayTransform(IrreversibleTransform):
     """
     HolidayTransform generates series that indicates holidays in given dataset.
 
-    In ``binary`` mode shows the presence of holiday in that day. In ``category`` mode shows the name of the holiday
-    with value "NO_HOLIDAY" reserved for days without holidays.
+    - In ``binary`` mode shows the presence of holiday in that day.
+    - In ``category`` mode shows the name of the holiday with value "NO_HOLIDAY" reserved for days without holidays.
+    - In the ``days_count mode``, calculate frequency of holidays.
```

Comment:
In documentation we should clarify with this frequencies this mode works and how exactly it works.